includes/constants/class-wc-stripe-payment-methods.php

```diff
@@ -35,6 +35,8 @@ class WC_Stripe_Payment_Methods {
 	const WECHAT_PAY        = 'wechat_pay';
 	const OC                = 'card'; // This is a special case for the Optimized Checkout
 
+	const LEGACY_SEPA       = 'stripe_sepa'; // Sepa method identifier for the legacy checkout (now removed)
```

Comment:
Nit: should we make these constants explicitly public?
```suggestion
	public const LEGACY_SEPA       = 'stripe_sepa'; // Sepa method identifier for the legacy checkout (now removed)
```